clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleClusterProvider.groovy

```diff
@@ -106,7 +108,9 @@ class GoogleClusterProvider implements ClusterProvider<GoogleCluster.View> {
       Keys.getClusterKey(account, application, name),
       RelationshipCacheFilter.include(SERVER_GROUPS.ns))
 
-    return clusterData ? clusterFromCacheData(clusterData, includeDetails) : null
+    def allClusterInstanceKeys = cacheView.filterIdentifiers(INSTANCES.ns, Keys.getInstanceKey(account, "*", "$name-*"))
```

Comment:
If you're doing this sort of name matching, can you generate the name (including `*`) using Frigga? It'll make it easier to remove later.